docs-src/go/how-to-use-worker-versioning-in-go.md

```diff
@@ -115,3 +115,22 @@ var yourActivityResult YourActivityResultType
 err := workflow.ExecuteActivity(ctx, YourActivityDefinition, yourActivityParam).Get(ctx, &yourActivityResult)
 // ...
 ```
+
+### Specifying Versioning for Continue-As-New
+
+When using the Continue-As-New, specify the versioning intent:
+
+1. Use the `WithWorkflowVersioningIntent` context modifier:
+   ```go
+   ctx = WithWorkflowVersioningIntent(ctx, temporal.VersioningIntentLatest)
```

Comment:
```suggestion
   ctx = workflow.WithWorkflowVersioningIntent(ctx, temporal.VersioningIntentDefault)
```
1. workflow code has to refer to the package name
2. it's called `VersioningIntentDefault`